src/parser/core/modules/SpeedmodConsts.js

```diff
@@ -0,0 +1,155 @@
+import STATUSES from 'data/STATUSES'
```

Comment:
This file is :blobknife:, but I realise lookup tables are basically the only accurate info we've got right now.

Looking forward to a proper formula to replace all this ;-;